README.md

```diff
@@ -34,101 +62,64 @@ make deploy
-the operational tasks.
+All these requests must be served via HTTPS and a CA must be provided to ensure that
+the API Server is communicating with the right client. Capsule upon installation is setting its custom Certificate Authority as a client certificate as well, updating all the required resources to minimize the operational tasks.
 
 ## Tenant users
+Each tenant comes with a delegated user acting as the tenant admin. In the Capsule jargon, this user is called the _Tenant Owner_. Other users can operate inside a tenant with different levels of permissions and authorizations assigned directly by the Tenant owner.
 
-All Tenant owner needs to be granted with a X.509 certificate with
-`capsule.clastix.io` as _Organization_.
-
-> the [hack/create-user.sh](hack/create-user.sh) can help you setting up a
-> dummy kubeconfig
->
-> ```
-> #. /create-user.sh alice oil
-> creating certs in TMPDIR /tmp/tmp.4CLgpuime3 
-> Generating RSA private key, 2048 bit long modulus (2 primes)
-> ............+++++
-> ........................+++++
-> e is 65537 (0x010001)
-> certificatesigningrequest.certificates.k8s.io/alice-oil created
-> certificatesigningrequest.certificates.k8s.io/alice-oil approved
-> kubeconfig file is: alice-oil.kubeconfig
-> to use it as alice export KUBECONFIG=alice-oil.kubeconfig
-> ```
+Capsule does not care about the authentication strategy used in the cluster and all the Kubernetes methods of [authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) are supported. The only requirement to use Capsule is to assign tenant users to the `capsule.clastix.io` group.
 
-## How to create a Tenant
+Assignment to a group depends on the authentication strategy in your cluster. For example, users authenticated through a _X.509_ certificate must have `capsule.clastix.io` as _Organization_: `-subj "/CN=${USER}/O=capsule.clastix.io"`
+
+Users authenticated through an _OIDC token_ must have
+
+```json
+...
+"users_groups": [
+    "/capsule.clastix.io",
+    "other_group"
+  ]
+```
+
+in their token.
+
+The [hack/create-user.sh](hack/create-user.sh) can help you set up a dummy `kubeconfig` for the `alice` user acting as owner of a tenant called `oil`
+
+```bash
+~/capsule$ ./hack/create-user.sh alice oil
+creating certs in TMPDIR /tmp/tmp.4CLgpuime3 
+Generating RSA private key, 2048 bit long modulus (2 primes)
+............+++++
+........................+++++
+e is 65537 (0x010001)
+certificatesigningrequest.certificates.k8s.io/alice-oil created
+certificatesigningrequest.certificates.k8s.io/alice-oil approved
+kubeconfig file is: alice-oil.kubeconfig
+to use it as alice export KUBECONFIG=alice-oil.kubeconfig
+```
 
+## How to create a Tenant
 Use the [scaffold Tenant](config/samples/capsule_v1alpha1_tenant.yaml)
 and simply apply as Cluster Admin.
 
 ```
-# kubectl apply -f config/samples/capsule_v1alpha1_tenant.yaml
+~/capsule$ kubectl apply -f config/samples/capsule_v1alpha1_tenant.yaml
```

Comment:
honestly, I would go back to the `#` symbol rather than `~/capsule$`.

If `#` can be misunderstood, let's put just the `~` or `#` symbol.

Reply:
done